addons/func_godot/src/map/func_godot_map.gd

```diff
@@ -572,6 +583,9 @@ func build_entity_collision_shapes() -> void:
 		else:
 			func_godot.gather_entity_convex_collision_surfaces(entity_idx)
 		var entity_surfaces: Array = func_godot.fetch_surfaces(func_godot.surface_gatherer)
+		var metadata := func_godot.surface_gatherer.out_metadata
```

Comment:
Should be `var metadata: Dictionary = func_godot.surface_gatherer.out_metadata` to match style throughout func_godot_map.gd. Try to avoid `:=` typing when possible, try to be explicit.

I know, Core needs a lot of cleanup in this area.